examples/application/sandbox.py

```diff
@@ -0,0 +1,15 @@
+# import pyteal.ast.jsonref
```

Comment:
I think this file is not meant to be here?

Reply:
@jasonpaulos Thanks for catching - File that one under _oops_.  Removed via https://github.com/algorand/pyteal/pull/462/commits/e21e4632661ff7a93b89e6cebf5fdb0d08796608.